docs/pages/database-access/guides/elastic.mdx

```diff
@@ -0,0 +1,248 @@
+---
+title: Database Access with Elasticsearch
+description: How to configure Teleport Database Access with Elasticsearch.
+---
+
+<Details
+  title="Version warning"
+  opened={true}
+  scope={["oss", "enterprise"]}
+  scopeOnly={true}
+  min="10.3"
+>
+  Database access for Elasticsearch is available starting from Teleport `10.3`.
+</Details>
+
+This guide will help you to configure secured access to an Elasticsearch database using Teleport Dabatase Access.
+
+## Prerequisites
+
+- A self-hosted Elasticsearch database. Elastic cloud [does not support client certificates](https://www.elastic.co/guide/en/cloud/current/ec-restrictions.html#ec-restrictions-security).
+
+- A host where you will run the Teleport Database Service. Teleport version 10.3
+  or newer must be installed. <ScopedBlock scope={["oss", "enterprise"]}> This can also be the same instance of Teleport running your Auth and/or Proxy service(s).</ScopedBlock>
+
+  See [Installation](../../installation.mdx) for details.
+
+(!docs/pages/includes/user-client-prereqs.mdx!)
+
+## Step 1/5. Install and configure Teleport
+
+### Set up the Teleport Auth and Proxy Services
+
+(!docs/pages/includes/database-access/start-auth-proxy.mdx!)
+
+### Set up the Teleport Database Service
+
+(!docs/pages/includes/database-access/token.mdx!)
+
+Install Teleport on the host where you will run the Teleport Database Service. Make sure you've selected your installation type (OSS, Enterprise, Clo):
+
+(!docs/pages/includes/install-linux.mdx!)
+
+Start the Teleport Database Service, pointing the `--auth-server` flag to the
+address of your Teleport Proxy Service:
+
+<Tabs>
+<TabItem label="Standalone Binary">
+
+```code
+$ teleport db start \
+  --token=/tmp/token \
+  --auth-server=teleport.example.com:3080 \
+  --name=myelastic \
+  --protocol=redis \
+  --uri={ELASTICSEARCH_HOST}:9200 \
+  --labels=env=dev
+```
+
+</TabItem>
+<TabItem label="Config File">
+Add the following block to `teleport.yaml`
+```yaml
+db_service:
+  enabled: true
+  resources:
+  - labels:
+      "*": "*"
+  databases:
+  - name: myelastic
+    protocol: elasticsearch
+    uri: {ELASTICSEARCH_HOST}:9200
+    static_labels:
+      env: dev
+```
+
+Adjust for your environment, then start or restart Teleport.
+
+</TabItem>
+</Tabs>
+
+<Admonition type="note">
+
+The `--auth-server` flag must point to the Teleport cluster's Proxy Service
+endpoint because the Database Service always connects back to the cluster over a
+reverse tunnel.
+
+</Admonition>
+
+<Admonition type="tip">
+  You can start the Database Service using a configuration file instead of CLI flags.
+  See the [YAML reference](../reference/configuration.mdx) for details.
+</Admonition>
+
+## Step 2/5. Create or modify a Teleport user
+
+(!docs/pages/includes/database-access/create-user.mdx!)
+
+Alternately, you can edit an existing user with database access.
+
+1. As a Teleport user with the `editor` role, import a user's configuration to a local `yaml` file:
```

Comment:
This section isn't specific to Elasticsearch and we don't have it in other guides. There are also other ways to achieve access given general flexibility of Teleport RBAC. It also won't work for SSO users. 

It is important to ensure that user can access the database, but perhaps we can delegate this to another article that can discuss the problem in depth? This section could then be replaced with instructions for quick access check (`tsh db ls`) and a link to the aforementioned article.

Reply:
It looks like we have such a page, "Database Access Access Controls". Since a link to that page is at the bottom of this one, and with @ptgott's comment above, I think the best action is to remove this section, and possibly include a note in the `create-user.mdx` partial referencing it.